needed so the bins find libs from the wine libdir
--
diff --git a/configure.ac b/configure.ac
index 140f26a..02497c9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -741,6 +741,7 @@
                             [AS_VAR_APPEND([WINEPRELOADER_LDFLAGS],[" -static -Wl,-Ttext=0x7d7d00000000"])])
             ;;
         esac
+        WINE_TRY_CFLAGS([-Wl,-rpath,\\\$ORIGIN],[UNIXLDFLAGS="$UNIXLDFLAGS '-Wl,-rpath,\$\$ORIGIN'"])
         ;;
       *)
         case $HOST_ARCH in
